packages/cxl-ui/package.json

```diff
@@ -17,15 +17,16 @@
     "@pika/plugin-build-web": "^0.9.2",
     "@pika/plugin-bundle-web": "^0.9.2",
     "@pika/plugin-standard-pkg": "^0.9.2",
-    "@vaadin/vaadin-accordion": "^2.0.0-alpha5",
-    "@vaadin/vaadin-button": "^3.0.0-alpha1",
-    "@vaadin/vaadin-checkbox": "^3.0.0",
+    "@vaadin/accordion": "^23.1.0",
+    "@vaadin/button": "^23.1.0",
+    "@vaadin/checkbox": "^23.1.0",
+    "@vaadin/context-menu": "^23.1.0",
+    "@vaadin/dialog": "^23.1.0",
+    "@vaadin/notification": "^23.1.0",
+    "@vaadin/progress-bar": "^23.1.0",
+    "@vaadin/tabs": "^23.1.0",
     "@vaadin/vaadin-context-menu": "^5.0.0-alpha2",
```

Comment:
We should just refactor our vaadin-device-detector usage to their newer mixins. Solution seems to live at https://github.com/vaadin/web-components/blob/ad1ad3d947c605dbe8d50f72e8348ac1de19688e/packages/context-menu/src/vaadin-context-menu.js#L13 using `MediaQueryController`.

Reply:
I thought about this. I didn't have the headspace to take care of it :)

Reply:
They do use a `ControllerMixin` on top of `PolymerElement`, I wasn't sure if `cxl-app-layout` was ready for that.